tests/core/htmlparser/manual/preserveinlinestyling.html

```diff
@@ -0,0 +1,30 @@
+<div id="editor">
+	<p>
+		<span style="font-size: 16px"
+			><span style="font-family: Courier New, Courier, monospace"
+				>hello, world!<br /><br /><br /></span
+		></span>
+	</p>
+</div>
+
+<script>
+	// IE allows to set cursor position exactly in BR tag.
+	if ( CKEDITOR.env.ie ) {
+		bender.ignore();
+	}
```

Comment:
We could wrap the entire script in IIFE and return the bender.ignore(); to prevent further editor creation, like in https://github.com/ckeditor/ckeditor4/blob/master/tests/core/editable/manual/enterbr.html

Reply:
I don't really see a reason for that. There is just a single editor and the test you linked is rather an exception from our standard approach and a rule. Sometimes a developer wants to perform a manual test on an unsupported browser even if it shouldn't be considered a valid case if that test failed.

Reply:
😅  Actually this is a way I was taught to write manual tests ;p I'm not super 'on it' - but so far I tried to review each PR to use that early return. If I good remember it comes from some case when the editor tries to initialize despite the test should be ignored and some error was thrown - so the 'ignored' was not visible in the UI. So it looks like the test failed on some browser.

However, we can live without such construction 👍